### ISSUE TITLE
Tk_eh slightly different to original code

### ISSUE THREAD

**Issue by steven-murray** [alexander-mead/HMcode-old#1]
Hi,

Thanks for the great code! I've just been using it to compare to my own code, and found a couple of (very small) things.

Firstly, in the TK_eh function, the rd = .... line, should that end with /(1+zd), rather than just /zd?  It seems to be the former in http://background.uchicago.edu/~whu/transfer/tf_fit.c.

**Comment by alexander-mead** [comment]
Hi Steven,

Sorry for the delay in getting back to you, I've just got back from a long
holiday. Anyway, I'm very pleased that you are using the code and that it
is being scrutinised to such a high degree.

I was just looking at the original, published paper (
http://iopscience.iop.org/article/10.1086/305424/pdf) and in that the 'R'
function (equation 5; from which R_s and R_eq are found) seems to lack the
'1+z' and just has a 'z' instead. This is different to that from
http://background.uchicago.edu/~whu/transfer/tf_fit.c where R_equality has
a 'z' and R_drag has a '1+z'... so, I'm not sure which is correct? In any
case, I guess the difference this would make would be small. I did a simple
check and it peaks around 0.05%, so I'm not too worried.

Cheers,
Alex
